.travis.yml

```diff
@@ -4,13 +4,23 @@ node_js:
   - 0.10
   - 0.12
   - 4
+# NodeJS v4 requires gcc 4.8
 env:
-  - NODE_ENV=travis
+  - NODE_ENV=travis CXX="g++-4.8" CC="gcc-4.8"
 matrix:
```

Comment:
I don't think that's the reason for it, but rather node-inspector which is failing.